packages/react-mui-store-file-viewer/src/components/CreateNewShareButton/CreateSharesButton/ShareAuthenticatorField.tsx

```diff
@@ -29,266 +31,243 @@ interface FieldProps {
 }
 
 interface AuthenticatorValue {
+  shareType: string;
   authenticators: ObjectStore.ShareAuthenticator[];
   authenticatorType: string | undefined;
 }
 
 const UnControlledShareAuthenticatorField: FC<
   FieldComponentProps<FieldProps>
 > = ({ onValueChange, ...props }) => {
-  const { configurations } = useStoreFileViewerContext();
-
   const value: AuthenticatorValue = props.value;
   const theme = useTheme();
+  const { contentType, password, loading } = useCreateShareContext();
   const [authenticatorType, setAuthenticatorType] =
     useState<ObjectStore.ShareAuthenticatorType>(
       ObjectStore.ShareAuthenticatorType.Sms
     );
+  const minHeight =
+    value.shareType === "link" && contentType === "file"
+      ? "350px"
+      : value.shareType === "link"
+        ? "250px"
+        : "190px";
 
   const handleAuthenticatorTypeChange = (
-    event: React.MouseEvent<HTMLElement>,
+    event: React.MouseEvent<HTMLElement> | undefined,
     newType: string
   ) => {
     if (!newType) return;
     if (!!onValueChange) {
-      onValueChange({
-        ...value,
-        authenticatorType: newType as ObjectStore.ShareAuthenticatorType,
-      });
+      // onValueChange({
```

Comment:
Is this still meant to be commented out?

Reply:
No longer needed, removed.